assignments/greatnumbergame/templates/index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<title>Number Game</title>
+	<style>
+		#response{
+			display: none;
+		}
+		{%if session['response'] == "YOU GOT IT RIGHT"%}
+			#form{
+				display: none;
+			}
+			#response{
+				display: block;
+			}
+		{%endif%}
```

Comment:
Not bad. I'd prefer you define all your styles, for example: .show{} and .hide{} classes. Then, where you actually define the class below, you do your if-check logic and either display "class='show'" or "class='hide'".